tests/kabir_practice/matrix_multiplier.futil

```diff
@@ -0,0 +1,165 @@
+import "primitives/core.futil";
+import "primitives/binary_operators.futil";
+import "primitives/memories/comb.futil";
+
+component main() -> () {
+    cells{
+        //Memory blocks
+        @external m1 = comb_mem_d2(32, 3, 3, 32, 32);
+        @external m2 = comb_mem_d2(32, 3, 3, 32, 32);
+        @external result = comb_mem_d2(32, 3, 3, 32, 32);
```

Comment:
FYI, we are now asking folks to go for `seq` memories and not `comb` memories. More info here:
https://docs.calyxir.org/libraries/core.html#memories